stable/dynamic-gateway-service/templates/statefulset.yaml

```diff
@@ -65,6 +68,33 @@ spec:
 {{- if eq (.Values.datapower.env.enableTMS | default "off") "on" }}
             - name: tms
               mountPath: /drouter/ramdisk2/mnt/raid-volume/raid0/local
+{{- end }}
+{{- if .Values.datapower.adminUserSecret }}
+            # In order to replace the default admin credentials, the new credentials should be configured via Secret and `.Values.datapower.adminUserSecret` must be set to the name of 
```

Comment:
This comment block should be moved to values.yaml, or at least copied there. We also need to put an empty reference to `datapower.adminUserSecret` in the Values.yaml.